patches/third_party-widevine-cdm-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/widevine/cdm/BUILD.gn b/third_party/widevine/cdm/BUILD.gn
+index 6225a610f11b3c4b1eb694f4e9e8f42bca8ffe09..de07d02b3bbf0ee92317ed83459d8454da9f348d 100644
+--- a/third_party/widevine/cdm/BUILD.gn
++++ b/third_party/widevine/cdm/BUILD.gn
+@@ -74,6 +74,7 @@ if (bundle_widevine_cdm && enable_widevine_cdm_host_verification) {
+ 
+ copy("version_h") {
+   visibility = [ ":*" ]  # Depend on ":headers" instead.
++  visibility += brave_third_party_widevine_cdm_visibility
```

Comment:
we are using the wrong dep if we need this. See line above `# Depend on ":headers" instead.`

Reply:
Ah, I see. I don't remember why I added this. Now it's not needed anymore. Removed.